### ISSUE TITLE
`ReferenceError: status is not defined`

### ISSUE THREAD

**Issue by KishanBagaria** [getsentry/sentry-electron#250]
## Versions + Platform

- [x] SDK version - `@sentry/electron@v1.4.0`
- [x] Electron version - `electron@v9.1.0`
- [x] Platform - `macOS`

## Description

I don't have a demo. My hunch is this happens on an unhandled promise rejection and when the Sentry server returns a non-200 response:

<img width="532" alt="" src="https://user-images.githubusercontent.com/1093313/88233494-667ec980-cc95-11ea-9c52-c7965da72324.png">

```
ReferenceError: status is not defined
    at ClientRequest.<anonymous> (/Applications/MyApp.app/Contents/Resources/app/node_modules/@sentry/electron/dist/main/transports/net.js:46:25)
    at ClientRequest.emit (events.js:223:5)
    at ClientRequest.EventEmitter.emit (domain.js:475:20)
    at SimpleURLLoaderWrapper.<anonymous> (electron/js2c/browser_init.js:93:6152)
    at SimpleURLLoaderWrapper.emit (events.js:223:5)
    at SimpleURLLoaderWrapper.EventEmitter.emit (domain.js:475:20)
    at SimpleURLLoaderWrapper.topLevelDomainCallback (domain.js:130:23)
```

The relevant code in `node_modules/@sentry/electron/dist/main/transports/net.js`:
```
const req = electron_1.net.request(options);
req.on('error', reject);
req.on('response', (res) => {
    if (res.statusCode && res.statusCode >= 200 && res.statusCode < 300) {
        resolve({
            status: types_1.Status.fromHttpCode(res.statusCode),
        });
    }
    else {
        if (status === types_1.Status.RateLimit) { // this line
```

**Comment by HazAT** [comment]
Will take a look.

**Comment by HazAT** [comment]
Fixed in `1.5.1`
